### PR TITLE
UI/AppKit: Fix tab title icon and text layout

### PR DESCRIPTION
**Pull request by bestwnh**
Make the tab title icon and text vertical center
image

### CODE REVIEW

**Comment by bestwnh** [comment]
The new commit make the ladybird icon size match other favicon size
image

**Comment by AtkinsSJ** [comment]
Hi! That looks a lot better thanks. :^)

Please squash the two commits together, as the second just corrects the first. See [this video](https://www.youtube.com/watch?v=ElRzTuYln0M) for help on how to do that, or you can ask for help on Discord.

**Comment by bestwnh** [comment]
@AtkinsSJ Job done.

File: Ladybird/AppKit/UI/Tab.mm [hunk 1]
```diff
@@ -205,15 +207,19 @@ - (void)updateTabTitleAndFavicon
...
+    // Add a offset to make the icon vertical center, maybe have better solution
+    [favicon_attribute addAttribute:NSBaselineOffsetAttributeName
+                              value:@-2
+                              range:NSMakeRange(0, [favicon_attribute length])];
 
     // By default, the text attachment will be aligned to the bottom of the string. We have to manually
     // try to center it vertically.
-    // FIXME: Figure out a way to programmatically arrive at a good NSBaselineOffsetAttributeName. Using
-    //        half the distance between the font's line height and the height of the favicon produces a
-    //        value that results in the title being aligned too low still.
```
Comment (trflynn89) [inline Ladybird/AppKit/UI/Tab.mm:213]: Let's move this FIXME comment above the new `NSBaselineOffsetAttributeName` of -2 above, as we didn't really resolve it here, we just changed where we hard-code the offset.

**Review by trflynn89 — COMMENTED** [review]
Please use `UI/AppKit` for the commit category.
(When in doubt of what to use for a category, it's best to `git log` related files and copy what was used recently).

**Comment by bestwnh** [comment]
@trflynn89 Updated the category and change to a new way to center the icon and text without fixed value.

File: Ladybird/AppKit/UI/Tab.mm [hunk 2]
```diff
@@ -195,6 +195,10 @@ - (TabController*)tabController
 
 - (void)updateTabTitleAndFavicon
 {
+    CGFloat titleFontSize = 12;
+    CGFloat faviconSize = 16;
```
Comment (trflynn89) [inline Ladybird/AppKit/UI/Tab.mm:199]: Last comment - let's keep these in line with the coding style guide:
https://github.com/LadybirdBrowser/ladybird/blob/master/Documentation/CodingStyle.md#names

```c++
static constexpr CGFloat TITLE_FONT_SIZE = 12;
static constexpr CGFloat FAVICON_SIZE = 16;
```

**Review by trflynn89 — COMMENTED** [review]
Thanks! That's looking much better, one last comment on the latest patch about coding style.

**Review by trflynn89 — APPROVED** [review]

**Pull request merged**